app/assets/stylesheets/components/_error-message.scss

```
@@ -0,0 +1,26 @@
+.error-message {
+  @extend .card;
+
+  margin-bottom: 1rem;
```

```suggestion
  margin-bottom: rem(16px);
```
It's easier to understand.

---

Resolved in a89ff14